infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/OpenApiTypeDefinition.java

```diff
@@ -43,6 +43,10 @@ default boolean isEquivalentToDeserializableType(final DeserializableTypeDefinit
     return this == type;
   }
 
+  default boolean isEquivalentToDeserializableType(final SerializableTypeDefinition<?> type) {
```

Comment:
Just double checking... was this meant to be named `isEquivalentToSerializableType`?

Reply:
My bad, this isn't used anymore anyway. Removed it